tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -1227,7 +1219,7 @@ PyMethodDef variable_methods[] = {
   {"set_", castPyCFunctionWithKeywords(THPVariable_set_), METH_VARARGS | METH_KEYWORDS, NULL},
   {"short", castPyCFunctionWithKeywords(THPVariable_short), METH_VARARGS | METH_KEYWORDS, NULL},
   {"size", castPyCFunctionWithKeywords(THPVariable_size), METH_VARARGS | METH_KEYWORDS, NULL},
-  {"storage", THPVariable_storage, METH_NOARGS, NULL},
+  {"_storage", THPVariable_storage, METH_NOARGS, NULL},
```

Comment:
I renamed this so that I could override its behavior in `torch/_tensor.py` to wrap the `ByteStorage` with the appropriate `<type>Storage`. Is this an acceptable approach to override `Tensor.storage()`'s behavior?

Reply:
Yes, this seems good